src/Json.Schema.Validation.UnitTests/TestData/ComprehensiveInstanceFile-validation-expected.sarif

```diff
@@ -722,12 +722,11 @@
       ],
       "tool": {
         "driver": {
-          "name": "Microsoft.Json.Schema.Validation.Cli",
-          "organization": "Microsoft",
-          "product": "JSON Schema",
-          "fullName": "Microsoft.Json.Schema.Validation.Cli 1.1.3.0",
-          "version": "1.1.3.0",
-          "semanticVersion": "1.1.3",
+          "name": "testhost",
+          "product": "Microsoft.TestHost",
+          "fullName": "testhost 15.0.0.0",
+          "version": "15.0.0.0",
+          "semanticVersion": "15.0.0",
```

Comment:
all changes in this file is only: 
I use the BuildAndTest.ps1 generated latest Actual file and replaced the current expected.